### DISCUSSION TITLE
Automatically updates

### DISCUSSION THREAD

**Discussion by hbadi** [pyvista/pyvista#2612] [Q&A]
Hi,

Is there a way to automatically make an update when vtk/vtu files change ?

Thanks for helping.

**Comment by banesullivan** [comment]
No, there is not

**Comment by banesullivan** [comment]
You would have to do some sort of polling on the mtime of the file and then have a custom call back to rerun your workflow. This is way out of scope for PyVista.

**Comment by banesullivan** [comment]
I will say that if you need to do this, you could use VTK directly as its pipeline model is built for this. But PyVista abandoned the pipeline model in favor of chaining to be more "Pythonic"
